### PR TITLE
Blog: This Week In React #1

### PR DESCRIPTION
**Pull request by ericnakagawa**
First post for "This Week In React" a new weekly publication that features interesting projects, event updates, and interviews from the React community.

### CODE REVIEW

File: docs/_data/authors.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ chenglou:
 Daniel15:
   name: Daniel Lo Nigro
   url: http://dan.cx/
+ericnakagawa:
+  name: Eric Nakagawa
+  url: http://twitter.com/ericnakagawa
```
Comment (zpao) [inline docs/_data/authors.yml]: nit: https :)

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+**Ken:** My name is Ken Wheeler, I work for Formidable and I've lived in the Jersey Shore my entire life.
+
+**Eric:** What brought you to the React community?
+
+**Ken:** A while back I saw React and gave it a shot. It really lended itself to building cool things quickly, so we've basically been in love since.
+
+**Eric:** Why did you build [react-game-kit](https://github.com/FormidableLabs/react-game-kit)?
+
+**Ken: **One of the first things I built with React Native was a game. Next, I built a physics game. I originally wanted to write a lib to bridge SpriteKit via native modules, but wanted things to be cross platform. I also noticed some repetitive patterns in what I was doing, so I wanted to create a set of helpers that handled the noisy stuff, so that users could focus on their game logic. I then proposed some talks on the subject, and the looming deadlines really pushed me to get it done quick and properly. It might not be the best way to write a game, but at the end of the day you definitely have a game and it works. You aren't going to write call of duty with it, but you can certainly build some cool 2d stuff.
```
Comment (acdlite) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: Misplaced space here is causing bold issues

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+title: "This Week In React #1"
+author: ericnakagawa
+---
+
+_This Week In React is a weekly publication highlighting interesting projects and active members from the React community. [Sign up to the weekly newsletter.](http://eepurl.com/cgOjbX)_
+
+![Projects aimed at beginning React developers.](/react/img/blog/thisweekinreact/beginner.png)
+
+####📓 A Visual Intro to Redux & Relay####
```
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: Nit: this should be an h3, so only `###`. I think that should help make the rendered output a bit more readable.

formatting nit: space after the opening hashes (gets stripped, makes the markdown more readable). And remove the trailing ones.

File: docs/_posts/2016-09-21-this-week-in-react-1.md (hunk 2 above)
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: Formatting is broken here (Space in the wrong place)

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+__Meetups__
+
+- [Thursday, September 22, 2016 San Francisco @ Minted](https://www.meetup.com/ReactJS-San-Francisco/events/231776842/)
+- [Thursday, September 22, 2016 Netherlands @ FrontMen](https://www.meetup.com/ReactJS-NL/events/233099575/)
+- [Saturday, September 24, 2016 Bangalore @ Multunus](https://www.meetup.com/ReactJS-Bangalore/events/233685933/)
+- [Wednesday, September 28, 2016 Seattle @ Redfin](https://www.meetup.com/seattle-react-js/)
+- [Thursday, September 29, 2016 Israel @ Klarna](https://www.meetup.com/ReactJS-IL/events/234262147/)
+- [Monday, October 3, 2016 Barcelona @ Lodgify](https://www.meetup.com/ReactJS-Barcelona/events/233971585/)
+- [Tuesday, October 18, 2016, London @ Facebook](https://meetup.react.london/)
```
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: Formatting on these is pretty awkward. Maybe we can use a verb? Do we get a ton of value out of the location name? I get that it's nice promotion for the host but it kind of clutters this up.

> Tuesday, October 18, 2016 in London at Facebook.

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+_Do you know about a React event coming up in the next 2 months? [Check out this event.](mailto:community@reactjs.org?subject=React%20Event)_
```
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: "Check out this event" is a weird call to action. Probably want to say "Email us" or something.

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+_This Week In React is a weekly publication highlighting interesting projects and active members from the React community. [Sign up to the weekly newsletter.](http://eepurl.com/cgOjbX)_
+
+![Projects aimed at beginning React developers.](/react/img/blog/thisweekinreact/beginner.png)
+
+####📓 A Visual Intro to Redux & Relay####
+
+Lin Clark has developed a series of visual guides for learning Flux, Redux, and most recently released a four-part visual guide to Relay.
+
+
+Redux: [https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6](https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6)
```
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: I'm generally not a huge fan of just using the URL as the link. Can we do anything about that (here and for a few things below too)?

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+
+Relay: [https://code-cartoons.com/a-cartoon-intro-to-facebook-s-relay-part-1-3ec1a127bca5](https://code-cartoons.com/a-cartoon-intro-to-facebook-s-relay-part-1-3ec1a127bca5)
+
+####🎮 React Game Development ([react-game-kit](https://github.com/FormidableLabs/react-game-kit))####
+
+React Game Kit lets you build a game in React. It supports basic game engine functions like a game loop, sprites and tiles, and also supports a simple physic engine.
```
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: >  lets you build a game in React

Lets you build *games*

Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: > simple physic engine

physics

**Comment by Aweary** [comment]
Is there some discussion somewhere I can reference regarding adding this new series to the blog?

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+![Projects aimed at intermediate React developers.](/react/img/blog/thisweekinreact/intermediate.png)
+
+####👾 Animations for Games ([BoxArt](https://github.com/boxart/boxart))####
+
+Animation is a core part of game development. The folks at Bocoup wrote up a guide for integrating Boxart, their library that better supports animations that a game may need. The guide demonstrates adding animations to a simple tile matching game. [https://bocoup.com/weblog/animating-react-elements-with-boxart](https://bocoup.com/weblog/animating-react-elements-with-boxart)
+
+####🎹 Music in React ([react-music](https://github.com/FormidableLabs/react-music))####
+
+The react-music project allows you produce music in JSX. The project supports setting tempo, managing instruments, and applying effects and other things you would expect to find in a sequencer/sampler. [https://formidable.com/blog/2016/08/22/make-dope-beats-with-reactjs/](https://formidable.com/blog/2016/08/22/make-dope-beats-with-reactjs/)
```
Comment (zpao) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: > allows you produce music in JSX

allows you *to* produce music *in React with* JSX

File: docs/_posts/2016-09-21-this-week-in-react-1.md (hunk 6 above)
Comment (acdlite) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: I suggest

```
- [A Cartoon Intro to Redux](https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6)
- [A Cartoon Intro to Relay](https://code-cartoons.com/a-cartoon-intro-to-facebook-s-relay-part-1-3ec1a127bca5)
```

**Comment by zpao** [comment]
@Aweary - no, this is coming about from after some chats with internal developer advocacy people :) A bit of a revival of sorts of the community roundups we used to do.

File: docs/_posts/2016-09-21-this-week-in-react-1.md (hunk 3 above)
Comment (ericnakagawa) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: The font size for h3 is massive relative to body copy.

File: docs/_posts/2016-09-21-this-week-in-react-1.md (hunk 6 above)
Comment (ericnakagawa) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: This is great feedback. I considered switching to text links. Thanks @zpao and @acdlite

**Comment by spicyj** [comment]
cc @facebook/react-core for all new blog posts

File: docs/_posts/2016-09-21-this-week-in-react-1.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+- [Thursday, September 22, 2016 - San Francisco at Minted](https://www.meetup.com/ReactJS-San-Francisco/events/231776842/)
+- [Thursday, September 22, 2016 - Netherlands at FrontMen](https://www.meetup.com/ReactJS-NL/events/233099575/)
+- [Saturday, September 24, 2016 - Bangalore at Multunus](https://www.meetup.com/ReactJS-Bangalore/events/233685933/)
+- [Wednesday, September 28, 2016 - Seattle at Redfin](https://www.meetup.com/seattle-react-js/)
+- [Thursday, September 29, 2016 - Israel at Klarna](https://www.meetup.com/ReactJS-IL/events/234262147/)
+- [Monday, October 3, 2016 - Barcelona at Lodgify](https://www.meetup.com/ReactJS-Barcelona/events/233971585/)
+- [Tuesday, October 18, 2016 - London at Facebook](https://meetup.react.london/)
+
+_Do you know about a React event coming up in the next 2 months? [Email us.](mailto:community@reactjs.org?subject=React%20Event)_
```
Comment (spicyj) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: This email doesn't work, right? We don't have MX records on reactjs.org as far as I can tell.

Comment (ericnakagawa) [inline docs/_posts/2016-09-21-this-week-in-react-1.md]: @spicyj community@reactjs.org now forwards to a support tool called HelpScout. That allows managing inbound emails. Advocacy team will review/respond when people write in to provide feedback/suggestions.

**Comment by vjeux** [comment]
For those who are watching at home, this has been published on medium: https://thisweekinreact.com/this-week-in-react-1-57001717c330#.dfgjd7vdo

**Pull request closed without merging**